### DISCUSSION TITLE
bottom card и диалоговое окно в ModalCard

### DISCUSSION THREAD

**Discussion by Ninjafio** [VKCOM/VKUI#8072] [Q&A] [answered]
Существует ли возможность при ширине окна <= 768 переключить режим модальной карточки с bottom card на диалоговое окно? Если нет то почему?

**Comment by inomdzhon** [comment, marked as answer]
в `<= v7.0.0` можно обернуть в [AdaptivityProvider](https://vkcom.github.io/VKUI/6.7.4/#/AdaptivityProvider)

```tsx
<AdaptivityProvider viewWidth={ViewWidth.SMALL_TABLET} hasPointer>
  <ModalRoot activeModal="x">
    <ModalCard id="x"> ... </ModalCard>
  </ModalRoot>
</AdaptivityProvider>
```

в `v7.0.0` эта возможность потерялась – исправим
